Program.cs

```diff
@@ -1,7 +1,37 @@
-﻿namespace Training {
+﻿// --------------------------------------------------------------------------------------------------
+// Training~A training program for new joinees at metamation,Batch-July 2023
+// Copyright(c) Metamation India
+// --------------------------------------------------------------------------------------------------
+// Program.cs
+// Function to group all anagram groups in the dictionary of words by using linq operators
+// Load the Words dictionary from the source
+// Find anagrams for all words by using orderby operator and take only words with atleast one anagram
+// Arrange the groups by descending order of anagram count
+// Display all the anagram groups
+// --------------------------------------------------------------------------------------------------
+using System.Reflection;
+
+namespace Training {
    internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+
+      #region Methods--------
+      public static void Main (string[] args) {
+         string[] input = LoadStrings ("dict-5.txt");
+         var groupedAnagrams = input
+            .GroupBy (word => string.Concat (word.OrderBy (c => c))) //Create groups of anagrams
+            .Where (group => group.Count () > 1).OrderByDescending (group => group.Count ()); //Remove words without anagrams and order by descending order
+         foreach (var item in groupedAnagrams) // Prints the anagrams with their count

```

Comment:
```suggestion
         foreach (var item in groupedAnagrams) //  Prints the anagrams with their count

```
Always leave a space after double slashes in the comment